yarn.lock

```diff
@@ -25,20 +25,35 @@
   dependencies:
     regenerator-runtime "^0.13.2"
 
-"@electron/docs-parser@^0.4.1", "@electron/docs-parser@^0.4.2":
-  version "0.4.2"
-  resolved "https://registry.yarnpkg.com/@electron/docs-parser/-/docs-parser-0.4.2.tgz#1d5feb0376363435b2f6833b60271ee90c26f952"
-  integrity sha512-BdEW2So0Qg7lxIET9q84DceavC1v/EyYF9PXU8vRRiwFebGPyEaJS4uqxhStAxmZslQAT4JjQc9jTd12IbG6BQ==
+"@electron/docs-parser@^0.4.1":
+  version "0.4.4"
+  resolved "https://registry.yarnpkg.com/@electron/docs-parser/-/docs-parser-0.4.4.tgz#4b22be74aa9073315241d2d4b06b22e7ff487106"
+  integrity sha512-EGQMXBIkW6JtwaxbmuWrmJivmrR420dX1cEy5I5Az71aVIz/E4gxlLmlT5L/jOzWwJ+1rbpri9LKVWPl0Bnasw==
   dependencies:
-    "@types/markdown-it" "^0.0.7"
+    "@types/markdown-it" "^0.0.9"
     chai "^4.2.0"
-    chalk "^2.4.2"
-    fs-extra "^7.0.1"
+    chalk "^3.0.0"
+    fs-extra "^8.1.0"
     lodash.camelcase "^4.3.0"
-    markdown-it "^8.4.2"
+    markdown-it "^10.0.0"
     minimist "^1.2.0"
-    ora "^3.4.0"
-    pretty-ms "^5.0.0"
+    ora "^4.0.3"
+    pretty-ms "^5.1.0"
+
+"@electron/docs-parser@^0.7.2":
```

Comment:
we probably shouldn't be depending on two different versions of docs-parser..?

Reply:
I'm going to address this in a follow up, it looks awkward but doesn't actually negatively impact anything